libs/database-entity/src/dto.rs

```diff
@@ -481,4 +481,35 @@ pub struct AFWorkspaceMember {
   pub avatar_url: Option<String>,
 }
 
-// pub type AFBlobMetadata = AFBlobMetadataRow;
+#[derive(Deserialize, Serialize, Debug)]
+pub struct AFWorkspaceInvitation {
+  pub invite_id: Uuid,
+  pub workspace_id: Uuid,
+  pub workspace_name: Option<String>,
+  pub inviter_email: Option<String>,
+  pub inviter_name: Option<String>,
+  pub status: AFWorkspaceInvitationStatus,
+  pub updated_at: DateTime<Utc>,
+}
+
+#[derive(Serialize, Deserialize, Eq, PartialEq, Debug, Clone)]
+#[repr(i16)]
```

Comment:
how about repr(u8) ?

Reply:
I have tried, smallest integer postgres can support is 16 bits